requirements/windows.txt

```diff
@@ -4,7 +4,7 @@
 
 pywin32>=305
 wmi>=1.5.1
-pythonnet>=2.5.2
+pythonnet>=3.0.1
```

Comment:
It seems like there are multiple things occuring in this PR.
- Adding MSI build scripts
- Updating pythonnet version. 
Can we break the PRs out so we don't have one big PR?

Reply:
The pythonnet PR has been merged and this rebased.